### PR TITLE
ovnk: get kubeconfig dir from microshift config

### PR DESCRIPTION
**Pull request by zshi-redhat**
Signed-off-by: Zenghui Shi <zshi@redhat.com>

### CODE REVIEW

File: pkg/assets/bindata.go [hunk 1]
```diff
@@ -16,7 +16,9 @@
 // assets/components/openshift-dns/dns/service-account.yaml
 // assets/components/openshift-dns/dns/service.yaml
 // assets/components/openshift-dns/node-resolver/daemonset.yaml
+// assets/components/openshift-dns/node-resolver/daemonset.yaml.tmpl
```
Comment (mangelajo) [inline pkg/assets/bindata.go:19]: Why is it adding this file here? seems unrelated to the patch

File: pkg/assets/bindata.go [hunk 2]
```diff
@@ -852,6 +854,93 @@ func assetsComponentsOpenshiftDnsNodeResolverDaemonsetYaml() (*asset, error) {
 	return a, nil
 }
 
+var _assetsComponentsOpenshiftDnsNodeResolverDaemonsetYamlTmpl = []byte(`apiVersion: apps/v1
```
Comment (mangelajo) [inline pkg/assets/bindata.go:857]: same

File: pkg/assets/bindata.go [hunk 3]
```diff
@@ -874,6 +963,80 @@ func assetsComponentsOpenshiftDnsNodeResolverServiceAccountYaml() (*asset, error
 	return a, nil
 }
 
+var _assetsComponentsOpenshiftDnsNodeResolverUpdateNodeResolverSh = []byte(`#!/bin/bash
```
Comment (mangelajo) [inline pkg/assets/bindata.go:966]: same

**Comment by mangelajo** [comment]
can you run the update with the exclude you proposed on the other PR ?

**Comment by zshi-redhat** [comment]
> can you run the update with the exclude you proposed on the other PR ?

done.

**Comment by zshi-redhat** [comment]
Rebased against master.

**Comment by zshi-redhat** [comment]
@mangelajo PTAL

**Review by mangelajo — APPROVED** [review]
/lgtm
